### PR TITLE
(slv-81) gem_of should package, deliver rake tasks

### PR DESCRIPTION
**Pull request by er0ck**

### CODE REVIEW

File: lib/gem_of/rake_tasks.rb [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+require "roodi_task"
+require "rubycritic/rake_task"
+
+module GemOf
+  # a bunch of instances of other rake tasks
+  # @example create an instance of me in your rakefile
+  #   GemOf::RakeTasks.new
+  # @api public
+  class RakeTasks
+    include Rake::DSL if defined? Rake::DSL
```
Comment (samwoods1) [inline lib/gem_of/rake_tasks.rb]: If Rake::DSL isn't defined wouldn't everything else blow up? Should we just throw here instead?

File: Rakefile [hunk 2]
```diff
@@ -1,138 +1,4 @@
-require "fileutils"
-require "rototiller"
-require "yard"
-require "rubocop/rake_task"
-require "flog_task"
-require "flay_task"
-require "roodi_task"
-require "rubycritic/rake_task"
+# FIXME: this might not work when specifying the rakefile from another path
```
Comment (samwoods1) [inline Rakefile]: You could check to see if the directory exists, and if not recursively go up a directory and try again?

File: lib/gem_of/rake_tasks.rb (hunk 1 above)
Comment (er0ck) [reply lib/gem_of/rake_tasks.rb]: yeah.  this is copy pasta from somewhere.

File: Rakefile (hunk 2 above)
Comment (er0ck) [reply Rakefile]: actually.  the Gemfile breaks if you try to run rake in bundle exec from another dir.  
in here we could use `application.original_dir` to find the project root.  but not in the Gemfile.

i'm gonna remove the comment and we'll cross that bridge later

**Pull request merged**